docs/integrations/sources/zendesk-chat-migrations.md

```diff
@@ -0,0 +1,58 @@
+# Zendesk Chat Migration Guide
+
+## Upgrading to 1.0.0
+
+Now, subdoimain is a required property in the spec, The base url is now being updated to use subdomain as per the [official update](https://developer.zendesk.com/api-reference/live-chat/introduction/) from zendesk chat documentation
+
+### For Airbyte Open Source: Update the local connector image
+
+Airbyte Open Source users must manually update the connector image in their local registry before proceeding with the migration. To do so:
+
+1. Select **Settings** in the main navbar.
+    - Select **Sources**.
+2. Find Zendesk Chat in the list of connectors.
+
+:::note
+You will see two versions listed, the current in-use version and the latest version available.
+:::
+
+3. Select **Change** to update your OSS version to the latest available version.
+
+### For Airbyte Cloud: Update the connector version
+
+1. Select **Sources** in the main navbar.
+2. Select the instance of the connector you wish to upgrade.
+
+:::note
+Each instance of the connector must be updated separately. If you have created multiple instances of a connector, updating one will not affect the others.
+:::
+
+3. Select **Upgrade**
+    - Follow the prompt to confirm you are ready to upgrade to the new version.
+
+### Refresh affected schemas and data
```

Comment:
I don't think this section is required as the breaking change does not impact records schema.